### PR TITLE
Add an option to send bootstrap logs to Loki in IPI

### PR DESCRIPTION
**Pull request by vrutkovs**
This change would update bootstrap ignition to run promtail service on bootstrap node so that service/pods would be logged to Loki when `OPENSHIFT_INSTALL_PROMTAIL_ON_BOOTSTRAP` is set.

The setting is most useful for jobs which often fail on bootstrap, adding more information to log bundles.

[Example](https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22refId%22:%22A%22,%22expr%22:%22%7Binvoker%3D%5C%22openshift-internal-ci%2Frehearse-25709-periodic-ci-openshift-release-master-ci-4.11-e2e-aws-calico%2F1488104034614120448%5C%22%7D%20%7C%20unpack%22%7D%5D) - just the bootstrap part of calico aws install

### CODE REVIEW

**Comment by DennisPeriquet** [comment]
@vrutkovs can you please point to a log where you think it ran to serve as proof it's working? thx

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml [hunk 1]
```diff
@@ -8,13 +8,23 @@ ref:
     requests:
       cpu: 1000m
       memory: 2Gi
+  credentials:
+  - namespace: test-credentials
```
Comment (DennisPeriquet) [inline ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml:12]: Just a question for my own training ... is it true that all credentials reside in the `test-credentials` namespace? thx

Comment (vrutkovs) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml:12]: Most of them, yes - see https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml

**Comment by vrutkovs** [comment]
@DennisPeriquet see https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22refId%22:%22A%22,%22expr%22:%22%7Binvoker%3D%5C%22openshift-internal-ci%2Frehearse-25709-periodic-ci-openshift-release-master-ci-4.11-e2e-aws-calico%2F1488104034614120448%5C%22%7D%20%7C%20unpack%22%7D%5D

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml (hunk 1 above)
Comment (DennisPeriquet) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml:12]: thanks

**Pull request merged**